src/Shell/Task/TemplateTask.php

```diff
@@ -431,6 +446,8 @@ public function getOptionParser()
             'help' => 'The controller name if you have a controller that does not follow conventions.'
         ])->addOption('prefix', [
             'help' => 'The routing prefix to generate views for.',
+        ])->addOption('index-columns', [
+            'help' => 'Limit for the number of index columns (Default: 7)'
```

Comment:
Option should probably be named max-table-columns :)
I don't know if it does already but it should then also be respected by view:related tables.